starters/http/dotnet/README.md

```diff
@@ -101,9 +72,10 @@ content-type: application/json
 
 1) Open this folder in a new terminal
 2) Open VS Code by entering `code .` in the terminal
-3) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
-4) Press Run/Debug (F5) to run in the debugger
-5) Use same approach above to test using an HTTP REST client
+3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
+4) Add needed files to the **.vscode** folder by opening the command palette (`Crtl+Shift+P/Cmd+Shift+P` and selecting *"Azure Functions: Initialize project for use with VS Code"* (select `starters\http\dotnet\http` project when prompted to set a default)
```

Comment:
Something got munged here, where I see CTL SHIFT P twice, and two open ('s.

Reply:
fixed